### PR TITLE
update osx build docs

### PR DESCRIPTION
**Pull request by mmienko**
- Add Jetbrains IDE dir to gitignore
- Document how to verify Homebrew during OSX build setup

While the project builds for osx M2, developers might not have envrionments properly setup (I didn't becuase I think I installed Homebrew from a guide rather than official site).
Hopefully improved documentation can mitigate future mishaps. Inspired by https://github.com/bitcoin/bitcoin/issues/25556.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -150,3 +150,6 @@ dist/
 /guix-build-*
 
 /ci/scratch/
+
+# Jetbrains
+.idea
```
Comment (hebasto) [inline .gitignore:155]: This should be done in developer's personal environment, not in the repo.

Comment (mmienko) [reply .gitignore:155]: Ah yeah, there is a global gitignore. Thanks, will revert.

**Comment by fanquake** [comment]
> (I didn't becuase I think I installed Homebrew from a guide rather than official site).

Concept NACK - Thanks, however steps for how to (re-)install your package manager, are off-topic for these installation instructions, and not something we want to maintain. I'd suggest always reading official (up-to-date) documentation, than random, third-party guides.

**Comment by mmienko** [comment]
> Thanks, however steps for how to (re-)install your package manager...not something we want to maintain.

That's a fair point and I agree. Appreciate the review. If I see issues for macOS like this, I'll leave a comment there.

**Pull request closed without merging**